docs/tutorials/07_benchmark_program.ipynb

```diff
@@ -1,19 +1,20 @@
 {
  "cells": [
   {
+   "attachments": {},
    "cell_type": "markdown",
    "id": "604850c4-053a-4b78-bff2-f5f0c9250ed8",
    "metadata": {},
    "source": [
     "# Tutorial: benchmark program to stress test compute resources\n",
     "\n",
-    "In order to test compute resources we need to have benchmark program.\n",
+    "In order to test compute resources we need to have a benchmark program.\n",
```

Comment:
@psschwei @pacomf I had some doubts about the word `program` used here, I didn't translate to `NestedProgram` because from my perception is generic but I would like to know your opinion.

Reply:
I think `program` is probably fine here, but if you want to draw a _really_ sharp distinction you could always use `benchmarch` in place of `program` in this one (don't think that's really necessary, but just tossing it out there as an option)

Reply:
or even better, use `benchmark`  :laughing:

Reply:
you can maintain program there, no update it. Nested Program is the fancy/corporative name, but at the end it is a program (used like synonym).

Reply:
Let me know what you think now @pacomf @psschwei  😄

Reply:
yes, I'm looking at it right now!